GitVersion.yml

```diff
@@ -1,5 +1,5 @@
 assembly-versioning-scheme: Major
-next-version: 1.0.0
+next-version: 1.1.0
```

Comment:
We haven't released this yet, should we not leave it as 1.0.0 for now?

Reply:
u need this to start building beta nuget of that version

Reply:
We won't release this to public nuget ever. Why would we?

Reply:
Does this get IL-Repacked into the V6 and V5 SC-integration? Why does it not end up on the public NuGet?

Reply:
Yes it is repacked. Why does it have to? It's not public API.

Reply:
Please feel free to take over this pr

Reply:
We probably still need to update the version number, otherwise we might get the wrong cached version from MyGet. If it's getting repacked, it really doesn't matter what the version number is right?

Reply:
I'm sorry @mikeminutillo but I'm not following. I didn't say anything about not updating the version number? I just stated that it's not released to NuGet. We apply regular versioning in here. What am I missing?